packages/lib/src/test/behaviors/Ownable.ts

```diff
@@ -0,0 +1,59 @@
+import assertRevert from '../helpers/assertRevert';
+import assert from 'assert';
+
+// TS-TODO: should this be in test/behaviors/ instead of src/test/behaviors/?
+export default function shouldBehaveLikeOwnable(owner:string, anotherAccount:string) {
+
+  describe('owner', function() {
+    it('sets the creator as the owner of the contract', async function() {
+      const contractOwner:string = await this.ownable.owner();
+      assert.equal(contractOwner, owner);
+    });
+  });
+
+  describe('transferOwnership', function() {
+    describe('when the proposed owner is not the zero address', function() {
+      const newOwner = anotherAccount;
+
+      describe('when the sender is the owner', function() {
+        const from = owner;
+
+        it('transfers the ownership', async function() {
+
```

Comment:
nit: Let's avoid adding unnecessary empty lines. I know we don't have rules for this, but I also believe we've been pretty consistent so far :smile:

Reply:
Sure! It's something I do to navigate paragraphs with Vim. Sorry if I am not aware of current conventions since I am not that familiar with the code right now. My suggestion is that you configure TSLint so that we start enforcing this rule with it. Again, you could do it now or just add a note to the linter setttings.